src/legacy/ui/public/vis/editors/default/_agg_select.scss

```diff
@@ -1,7 +1,3 @@
 .visEditorAggSelect__helpLink {
```

Comment:
To handle the only class selector in a separate file looks excessive.
Could we move it to `src\legacy\ui\public\vis\editors\default\_agg.scss` ?

Reply:
I removed this class and used `EuiText` for adjusting text size.